compiler/one-cmds/one-import-tf

```diff
@@ -83,6 +83,13 @@ while [ "$#" -ne 0 ]; do
   esac
 done
 
+if [ -n ${INPUT_SHAPES} ] && [ ${TF_INTERFACE} = "--v2" ]; then
+  echo "Error: shape cannot be assigned to dynamic shape in v2 version."
+  echo ""
+  usage
+  exit 2
```

Comment:
I think it would be better to show a warning message and not end the program for backward compatibility.
- message something like, `If --v2 option is used, shape will be ignored`